lib/src/mistralai_client_dart_base.dart

```diff
@@ -1,7 +1,12 @@
+import 'dart:convert';
 import 'package:http/http.dart' as http;
 import 'package:http/retry.dart' as retry;
 import 'package:mistralai_client_dart/src/models/models.dart';
 
+class _MistralAPIEndpoints {
+  static const String listModels = '/v1/models';
+}
```

Comment:
This should be in its own file, ideally in some "data" layer of this SDK 😉

Reply:
At the moment I think it could be just as part of the `listModels` method. We won't reuse it anywhere atm. I would like to keep it as simple as possible for now.

Reply:
I wouldn't recommend such shortcuts because they often bite us back in the future 😉 Especially that we're not in a hurry right now, and have the time to plan and think ahead 🤔

But if you insist to keep it simple then please create an issue for refactoring it later and past the link to that issue in this conversation 😉

Reply:
I think it will be good practice to keep API endpoints as static fields, but as @KacperKluka wrote moving this class into a separate file will be a better option.  
Wdyt? @gawi151

Reply:
We discussed on a call to leave it as it is for now as it will be refactored as a part of https://github.com/nomtek/mistralai_client_dart/issues/13